tests/suites/config.js

```diff
@@ -15,6 +15,10 @@ module.exports = [
 		}
 	},
 	image: `${__dirname}/balena-image.docker`,
+	// This is the name of a folder or file containing extra artifacts needed for secure-boot provisioning
+	// E.g for the cm4-io-board the `secure-boot-msd` folder is required
+	// This artifact/artifact must be copied into the `suites` folder before running the test
+	sbArtifacts: process.env.SB_ARTIFACTS,
```

Comment:
Maybe let's generalize to `artifacts`? We can add specific instruction for secure boot in the commit message.